tests/assets/spa/dt/xhr-dt-crossorigin-load.html

```diff
@@ -13,12 +13,23 @@
   <body>
     <div>This page makes a cross-origin XHR call when loading.</div>
     <script type="text/javascript">
+      window.rawTimes = []
       var xhr = new XMLHttpRequest()
 
       var url = 'http://' + NREUM.info.beacon + '/dt/' + NREUM.info.licenseKey
 
       xhr.open('GET', url)
+      window.rawTimes.push(Math.floor(performance.now()))
       xhr.send()
+
+      for (var i = 0; i < 10; i++) {
```

Comment:
10 might be excessive. It's better to test an ajax that isn't immediately fired after the last so that the floored start diff isn't 0.